assembly/src/parsers/field_ops.rs

```diff
@@ -176,3 +176,12 @@ fn parse_bit_len_param(op: &Token, param_idx: usize) -> Result<u8, ParsingError>
         Err(ParsingError::invalid_param(op, param_idx))
     }
 }
+
+fn parse_felt(op: &Token) -> Result<Felt, ParsingError> {
+    Ok(Felt::new(parse_checked_param::<u64>(
+        op,
+        1,
+        0,
+        Felt::MODULUS,
+    )?))
+}
```

Comment:
The upper bound should be `Felt::MODULUS - 1` since `Felt::MODULU` is not a valid value for a `Felt`.

Also, minor nit: I'd prefer the name of the function to be something like `parse_imm_value()`.